invokeai/backend/model_management/convert_ckpt_to_diffusers.py

```diff
@@ -1202,9 +1214,19 @@ def load_pipeline_from_original_stable_diffusion_ckpt(
 
         unet.load_state_dict(converted_unet_checkpoint)
 
-        # Convert the VAE model, or use the one passed
-        if not vae:
+        # If a replacement VAE path was specified, we'll incorporate that into
+        # the checkpoint model and then convert it
+        if vae_path:
+            print(f"  | Converting VAE {vae_path}")
+            replace_checkpoint_vae(checkpoint,vae_path)
+        # otherwise we use the original VAE, provided that
+        # an externally loaded diffusers VAE was not passed
+        elif not vae:
             print("  | Using checkpoint model's original VAE")
+
+        if vae:
```

Comment:
Can the user specify both `vae` and `vae_path`?

Reply:
If vae is specified, then it takes precedence over vae_path.